src/SwedbankPay/Api/Client/Version.php

```diff
@@ -52,30 +52,38 @@ protected function getVersionFromEnvironment()
         }
 
         $composerPath = __DIR__ . '/../../../../composer.json';
-        $composer = $this->readJson($composerPath);
+        if (file_exists($composerPath)) {
+            $composer = $this->readJson($composerPath);
 
-        if (isset($composer['version'])) {
-            return $composer['version'];
+            if (isset($composer['version'])) {
+                return $composer['version'];
+            }
         }
 
         $composerLockPath = getcwd() . DIRECTORY_SEPARATOR . 'composer.lock';
-        $composerLock = $this->readJson($composerLockPath);
-
-        if (isset($composerLock['packages'])) {
-            $packages = $composerLock['packages'];
-            foreach ($packages as $package) {
-                if (!isset($package['name']) ||
-                    $package['name'] != "swedbank-pay/swedbank-pay-sdk-php") {
-                    continue;
-                }
+        if (!file_exists($composerLockPath)) {
+            $composerLockPath = __DIR__ . '/../../../../composer.lock';
+        }
+
+        if (file_exists($composerLockPath)) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The use of function file_exists() is discouraged](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=5056329)